mcmc_3step_all.cpp

```diff
@@ -317,7 +317,7 @@ std::pair<Sample,Real> perturb(const Sample &sample,
   Sample new_sample(new_prm(0), new_prm(1), new_prm(2), new_prm(3),
                     new_prm(4), static_cast<unsigned int>(new_prm(5)));
 
-  std::cout << "New sample: " << new_sample << "\n";
+  //std::cout << "New sample: " << new_sample << "\n";
```

Comment:
I would just generally delete these lines. You can always get them back from git by looking at the history.